paper/main.tex

```diff
@@ -97,12 +98,12 @@ \section{Background}
 game in 1950~\cite{Flood1958}. Following the computer tournaments of R. Axelrod in the
 1980's~\cite{Axelrod1980a, Axelrod1980b}, a strategy's performance in a round
 robin computer tournament became a common evaluation technique for newly designed
-strategies. Many tournaments have followed Axelrod's~\cite{Bendor1991, Beaufils1997, Stephens2002,
-Kendall2007, Stewart2012, Harper2017} and today more hundreds of strategies exist  % TODO Order these citations
+strategies. Many tournaments have followed Axelrod's~\cite{Beaufils1997, Bendor1991,
+Harper2017, Kendall2007, Stephens2002, Stewart2012} and today hundreds of more strategies exist
```

Comment:
more than ... Axelrod's initial set? Maybe leave out the more, or rephrase "and now the literature and various codebases contain hundreds of strategies"

Reply:
I like the rephrase. 

I wasn't sure if the original `more hundreds of strategies exist` was correct. It just read weird to be. I even try google translate to check 😆